bot/exts/evergreen/githubinfo.py

```diff
@@ -74,24 +83,76 @@ async def get_github_info(self, ctx: commands.Context, username: Optional[str])
 
                 embed.add_field(name="Followers",
                                 value=f"[{user_data['followers']}]({user_data['html_url']}?tab=followers)")
-                embed.add_field(name="\u200b", value="\u200b")
                 embed.add_field(name="Following",
                                 value=f"[{user_data['following']}]({user_data['html_url']}?tab=following)")
 
             embed.add_field(name="Public repos",
                             value=f"[{user_data['public_repos']}]({user_data['html_url']}?tab=repositories)")
-            embed.add_field(name="\u200b", value="\u200b")
 
             if user_data['type'] == "User":
                 embed.add_field(name="Gists", value=f"[{gists}](https://gist.github.com/{username})")
 
                 embed.add_field(name=f"Organization{'s' if len(orgs)!=1 else ''}",
                                 value=orgs_to_add if orgs else "No organizations")
-                embed.add_field(name="\u200b", value="\u200b")
             embed.add_field(name="Website", value=blog)
```

Comment:
Should do:

I know this is not part of your PR, but it would be good if you could change these lines to follow our style guide i.e. each argument should be on a separate line.

Reply:
Might as well run `black`

Reply:
> Might as well run `black`

Yeah go ahead!

Reply:
We don’t follow black’s style guide, you may end up reverting more changes than you save.

Reply:
Where are these style guidelines?

Reply:
You can find them under the contributing section of the website:

https://pythondiscord.com/pages/contributing/style-guide/